targets/mgyamlfmt/targets.go

```diff
@@ -26,14 +25,29 @@ func FormatYaml() error {
 	})
 }
 
+// PreserveEmptyLines adds a temporary #comment on each empty line in the provided byte array.
+// Returns a function that cleans up the temporary comments.
+func PreserveEmptyLines(src []byte) ([]byte, func(in []byte) []byte) {
```

Comment:
why does the cleanup need to be done as a callback closure? why not just a separate function?

Reply:
no strong opinion, just looks a little out of the ordinary

Reply:
Fixed